### PR TITLE
Make README useful 

### PR DESCRIPTION
**Pull request by toskrip**
Proposing some changes into the README file to make it list useful links and adding contributions section.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+Check out [LibreClinica Documentation](https://libreclinica.org/documentation) to learn how to install, validate and use the software.
+
+### Development
+
+In case of interest in contributing into the codebase, check out [LibreClinica Developer Documentation](https://libreclinica-docs.readthedocs.io) to learn how to set up the development workstation and understand the branching strategy that is used to control the contribution workflows. 
+
+### Contributions
+                          
+LibreClinica is open source project that values input from contributors:
+
```
Comment (oba-reliatec) [inline README.md:19]: For version 1.2 most notably

File: README.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+* Thomas Hillger, University Hospital RWTH Aachen
```
Comment (oba-reliatec) [inline README.md:20]: We should ask Thomas Hillger, whether other colleagues (Julia Bley) should be also mentioned.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
+### Getting Started
+
+Check out [LibreClinica Documentation](https://libreclinica.org/documentation) to learn how to install, validate and use the software.
+
+### Development
+
+In case of interest in contributing into the codebase, check out [LibreClinica Developer Documentation](https://libreclinica-docs.readthedocs.io) to learn how to set up the development workstation and understand the branching strategy that is used to control the contribution workflows. 
+
+### Contributions
+
```
Comment (oba-reliatec) [inline README.md:17]: What about adding a link to https://libreclinica.org/goals.html here?

File: README.md (hunk 1 above)
Comment (toskrip) [reply README.md:19]: would make sense in changelog

File: README.md (hunk 2 above)
Comment (toskrip) [reply README.md:20]: sure, if there have been more people involved, just let me know and I update readme

File: README.md (hunk 3 above)
Comment (toskrip) [reply README.md:17]: ok, I will add this as well

**Comment by oba-reliatec** [comment]
See commit 370ec13cf8b0b4ca49af98474d998af4f59b3e34 before
Thank you for the changes
If all contributors listed agree to be published like that, ready to merge from my view.

**Pull request merged**